gwpy/cli/cliproduct.py

```diff
@@ -46,6 +46,7 @@
 from ..timeseries import TimeSeriesDict
 from ..plot.gps import (GPS_SCALES, GPSTransform)
 from ..plot.tex import label_to_latex
+from ..segments import DataQualityFlag   # noqa: E402
```

Comment:
I don't think the `noqa` is needed here.

```suggestion
from ..segments import DataQualityFlag
```

Reply:
I thing the early set matplotlib back end is why but I'll confirm.  It was pasted.

Reply:
You're right.  fixed.